### PR TITLE
doc/monitoring: add min vers of apps in mon stack

### PR DESCRIPTION
**Pull request by zdover23**
https://tracker.ceph.com/issues/45447

This PR adds recommended versions of grafana and
prometheus and alert manager.

This PR is a second attempt at getting the information
in the following PR into the docs:
https://github.com/ceph/ceph/pull/46000/files

Himadri Maheshwari deserves the credit for the work
in this commit.

Signed-off-by: Zac Dover <zac.dover@gmail.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Comment by zdover23** [comment]
jenkins retest this

**Comment by zdover23** [comment]
jenkins test make check

File: monitoring/ceph-mixin/README.md [hunk 1]
```diff
@@ -31,13 +37,21 @@ alert rules that should provide a decent set of default alerts for a
 Ceph cluster. Just put this file in a place according to your Prometheus
 configuration (wherever the `rules` configuration stanza points).
 
+
+##### Recommended versions: 
+-prometheus v2.7.2
+
 #### SNMP
 Ceph provides a MIB (CEPH-PROMETHEUS-ALERT-MIB.txt) to support sending Prometheus
 alerts through to an SNMP management platform. The translation from Prometheus
```
Comment (anthonyeleven) [inline monitoring/ceph-mixin/README.md:46]: While we're here, I think "through" is superfluous.

**Review by anthonyeleven — APPROVED** [review]

**Comment by zdover23** [comment]
jenkins test dashboard cephadm

**Comment by zdover23** [comment]
jenkins test dashboard cephadm

**Comment by epuertat** [comment]
@zdover23 can you please update the versions according to this comment: https://github.com/ceph/ceph/pull/46000#pullrequestreview-1077285333

File: monitoring/ceph-mixin/README.md (hunk 1 above)
Comment (zdover23) [reply monitoring/ceph-mixin/README.md:54]: I think you're right. 

I've removed it.

**Comment by zdover23** [comment]
> @zdover23 can you please update the versions according to this comment: [#46000 (review)](https://github.com/ceph/ceph/pull/46000#pullrequestreview-1077285333)

@epuertat: I've updated this document with the defaults listed in PR#46000.

**Comment by zdover23** [comment]
@epuertat @yuriw : The "ceph windows tests" test is failing with this message:

> /home/jenkins-build/build/workspace/ceph-windows-pull-requests/ceph/src/include/denc.h:517:49: error: 'bit_cast' is not a member of 'std'; did you mean

Here's the source of that error message:

https://jenkins.ceph.com/job/ceph-windows-pull-requests/13486/console

When this test passes, this PR can be merged. I'll need to know the defaults (or minimums, as the case may be) for the Pacific release in order to backport this accurately.

**Comment by epuertat** [comment]
> > @zdover23 can you please update the versions according to this comment: [#46000 (review)](https://github.com/ceph/ceph/pull/46000#pullrequestreview-1077285333)
> 
> @epuertat: I've updated this document with the defaults listed in PR#46000.

Thank you very much!

> When this test passes, this PR can be merged. I'll need to know the defaults (or minimums, as the case may be) for the Pacific release in order to backport this accurately.

Thery're the same:
https://github.com/ceph/ceph/blob/7b4e839796c28befa407d93d313da34fd130e786/src/cephadm/cephadm#L54-L57

**Comment by zdover23** [comment]
> > > @zdover23 can you please update the versions according to this comment: [#46000 (review)](https://github.com/ceph/ceph/pull/46000#pullrequestreview-1077285333)
> > 
> > 
> > @epuertat: I've updated this document with the defaults listed in PR#46000.
> 
> Thank you very much!
> 
> > When this test passes, this PR can be merged. I'll need to know the defaults (or minimums, as the case may be) for the Pacific release in order to backport this accurately.
> 
> Thery're the same:
> 
> https://github.com/ceph/ceph/blob/7b4e839796c28befa407d93d313da34fd130e786/src/cephadm/cephadm#L54-L57

@epuertat 

Do you understand these failed checks? I've been asking people elsewhere about these for a couple of days, but I haven't gotten any response.

**Comment by epuertat** [comment]
jenkins test make check

**Comment by epuertat** [comment]
@zdover23 no clue, I'm just retriggering the make check, which is the only mandatory one that's failing. About the other ones, if this does only touch docs, it's 100% an unrelated issue.

**Review by epuertat — APPROVED** [review]

**Comment by zdover23** [comment]
apt-get error

 Failed to fetch https://3.chacra.ceph.com/r/cortx-motr/master/39f89fa1c6945040433a913f2687c4b4e6cbeb3f/ubuntu/jammy/flavors/default/dists/jammy/main/binary-i386/Packages  404  Not Found [IP: 158.69.93.173 443]

**Comment by zdover23** [comment]
jenkins retest this please

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/48062 - Pacific backport
https://github.com/ceph/ceph/pull/48063 - Quincy backport

**Pull request merged**